packages/calculated-fields-bundle/src/lib/parser/grammar.pegjs

```diff
@@ -88,7 +88,7 @@ FunctionCall = name:FunctionName "(" args:Arguments ")" { return { type: "Functi
 
 Arguments = empty:(","*) first:Expression? rest:("," _ Expression)* { return buildArgumentList(empty, first, rest) }
 
-FunctionName = [A-Z]+ { return text() }
+FunctionName = [A-Z0-9]+ { return text() }
```

Comment:
Nit: the grammar should probably be 

```suggestion
FunctionName = [A-Z][A-Z0-9]* { return text() }
```

otherwise it will allow functions that start with a digit, which is not desirable